src/sentry/auth/providers/github/client.py

```diff
@@ -31,17 +32,17 @@ def _request(self, path, access_token):
             raise GitHubApiError(req.content, status=req.status_code)
         return json.loads(req.content)
 
-    def get_org_list(self, access_token):
-        return self._request("/user/orgs", access_token)
+    def get_org_list(self):
+        return self._request("/user/orgs", self.access_token)
```

Comment:
@slohmes Does `_request` not only have one argument? This seems like it would blow up.